locations/spiders/valora.py

```diff
@@ -0,0 +1,217 @@
+import re
+import scrapy
+from locations.items import GeojsonPointItem
+
+
+class ValoraSpider(scrapy.Spider):
+    name = "valora"
+    allowed_domains = ["api.valora-stores.com"]
+    key = "avecHnbG6Tr"
+    start_urls = [
+        "https://api.valora-stores.com/_index.php?apiv=2.0.9&a=data&"
+        + f"ns=storefinder&l=de&key={key}",
+    ]
+
+    brands = {
+        # For most Valora brands, we take tags such as "shop=bakery"
+        # from the OpenStreetMap Name Suggestion Index (NSI).
+        "avec": ("Avec", "Q103863974", {}),
+        "Back-Factory": ("Back-Factory", "Q21200483", {}),
+        "Backwerk": ("BackWerk", "Q798298", {}),
+        "Brezelkönig": ("Brezelkönig", "Q111728604", {}),
+        "Cigo": ("Cigo", "Q113290782", {}),
+        "Ditsch": ("Ditsch", "Q911573", {}),
+        "ServiceStoreDB": ("ServiceStore DB", "Q84482517", {}),
+        "Spettacolo": ("Caffè Spettacolo", "Q111728781", {}),
+        "U-Store": ("U-Store", "Q113290511", {}),
+        # However, the AllThePlaces pipeline does not apply any NSI rules
+        # if multiple rules match with conflicting tags. Perhaps the NSI data
+        # is buggy when there’s such a conflict; anyhow, by making an explicit
+        # decision here, we can work around this.
+        # See https://github.com/osmlab/name-suggestion-index/issues/7400.
+        "k kiosk": ("k kiosk", "Q60381703", {"shop": "kiosk;newsagent"}),
+        "Press & Books": ("Press & Books", "Q100407277", {"shop": "books;newsagent"}),
```

Comment:
I think I'd prefer you to pick one shop type for each of these and use it via [categories](https://github.com/alltheplaces/alltheplaces/blob/master/locations/categories.py).